addons/sale_management/models/sale_order_template.py

```diff
@@ -24,6 +27,7 @@ def _get_default_require_payment(self):
         help='Number of days for the validity date computation of the quotation')
     require_signature = fields.Boolean('Online Signature', default=_get_default_require_signature, help='Request a online signature to the customer in order to confirm orders automatically.')
     require_payment = fields.Boolean('Online Payment', default=_get_default_require_payment, help='Request an online payment to the customer in order to confirm orders automatically.')
+    required_payment_percentage = fields.Float('Payment Amount', default=_default_required_payment_percentage, help='Percentage of payment requested to the customer in order to confirm orders automatically.')
```

Comment:
Note: just update fields accordingly of other fields (require_confirmation_percentage if this name is chosen for example).